src/containers/ConversionForm/index.tsx

```diff
@@ -167,27 +175,41 @@ const ConversionForm = ({
         return <span className={styles.renderedFileName}>{fileName}</span>;
     };
 
+    const getErrorModal = () => {
+        switch (conversionError) {
+            case ConversionError.SERVER_ERROR:
+                return (
+                    <ConversionServerErrorModal closeModal={closeErrorModal} />
+                );
+            case ConversionError.FILE_TYPE_ERROR:
+                return (
+                    <ConversionFileErrorModal
+                        closeModal={closeErrorModal}
+                        engineType={conversionProcessingData.engineType}
+                    />
+                );
+            case ConversionError.FILE_SIZE_ERROR:
+                return (
+                    <ConversionFileSizeErrorModal
```

Comment:
are these modals all super different from each other? it seems like you could just send in a message instead of making a whole new component for each one

Reply:
Not super different no, could definitely consolidate to one component. The original designs showed some slight differences that might not be relevant since the Figma SSOT came out, and there are couple prop differences, nothing major.

Reply:
the one prop difference seems to be the engine type which could just be part of the generic `errorMessage` prop. especially considering they all have identical style sheets I would definitely combine them into one